pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -299,6 +298,19 @@ func (c *Reconciler) reconcileBuild(ctx context.Context, rev *v1alpha1.Revision)
 	return nil
 }
 
+func (c *Reconciler) reconcileDigest(ctx context.Context, rev *v1alpha1.Revision) error {
+	if rev.Spec.Container.Image != "" && rev.Status.ImageDigest == "" {
```

Comment:
I'd rotate this to:
```go
if rev.Status.ImageDigest != "" {
  return nil
}
```

... and drop the `rev.Spec.Container.Image` check, since it cannot be empty.

Reply:
Fixed in latest commits